### DISCUSSION TITLE
Issues with moisture sensor

### DISCUSSION THREAD

**Discussion by traverseda** [STEMinds/STEMinds-Docs#4] [General]
My moisture sensor seemed to work fine to start with, but now I'm consistently getting raw values in the 21-60 range from the ADC. I coated the upper part of the board in clear nail polish to try to make it water proof, it worked alright after that but I had a bit of flooding and it did end up mostly submerged (watch out for siphon effects). I don't see any damage on the board, but none the less it doesn't seem to be working.

Any thoughts?

**Comment by Zontex** [comment]
If the circuit touched the water it's probably dead. Another solution
besides nail polish can be hot glue gun. We made limited edition water
proof sensor exactly for that, probably should bring it back to market ...

**Comment by traverseda** [comment]
That's unfortunate, I'd hoped the nail polish would give it at least some cursory protection but I guess I missed a spot or something. I suppose it gives me an excuse to get the expansion board though, as I'm not paying the $18 usd shipping on a $5 usd item.

**Reply by Zontex** [reply]
> That's unfortunate, I'd hoped the nail polish would give it at least some cursory protection but I guess I missed a spot or something. I suppose it gives me an excuse to get the expansion board though, as I'm not paying the $18 usd shipping on a $5 usd item.

I'll give you a little secret. We are planning to bring back the limited edition water-proof sensor, this time we will make it by ourselves :)

**Comment by Zontex** [comment]
I can assure you it will worth it, the expansion is definitely a must !
